src/components/pages/Login/index.tsx

```diff
@@ -96,16 +102,17 @@ const Login = () => {
                 <Typography variant="h3" paragraph>
                     {translated.title}
                 </Typography>
-                <form onSubmit={handleSubmit}>
+                <form onSubmit={isActive ? handleSubmit:()=>{}}>
                     <Flex className={classes.inputWrapper} flexDirection="column">
-                        <TextField style={{ marginBottom: "15px" }} id="email-input" value={emailInput} onChange={handleInputChange("email")} type="email" label="E-Mail" variant="outlined" />
-                        <TextField style={{ marginBottom: "15px" }} id="password-input" value={passwordInput} onChange={handleInputChange("password")} type="password" label="Password" variant="outlined" />
+                        <TextField style={{ marginBottom: "15px"}} id="email-input" value={emailInput}onChange={isActive ? handleInputChange("email"):()=>{}} type="email" label="E-Mail" variant="outlined" />
+                        <TextField style={{ marginBottom: "15px"}} id="password-input" value={passwordInput} onChange={isActive ? handleInputChange("password"):()=>{}} type="password" label="Password" variant="outlined" />
                     </Flex>
 
                     {errorMsg && <Alert style={{margin: "10px 0"}} severity="error">
                         <AlertTitle>Upps...</AlertTitle>
                         {errorMsg}
                     </Alert>}
+                    {isActive ? null : <p>Loading...</p>}
```

Comment:
You could use the [CircularProgress](https://material-ui.com/api/circular-progress/) here instead of a raw text. Also if you add text, please wrap it with [Typography](https://material-ui.com/components/typography/#typography) component.

```suggestion
                    { isActive && <CircularProgress /> }
```